doc/admin-guide/files/records.config.en.rst

```diff
@@ -3721,6 +3734,240 @@ HTTP/2 Configuration
    |TS| gracefully closes connections that have stream error rates above this
    setting by sending GOAWAY frames.
 
+HTTP/3 Configuration
+====================
+
+There is no configuration on this release.
```

Comment:
Do you ignore `proxy.config.http3.*` intentionally?

Reply:
Yes. I don’t remember the exact reason, but they didn’t seem like something we want to show to users at the moment. I can add them if you want.

Reply:
These configs are knobs for HTTP/3 SETTINGS Parameters. It looks very similar to what we have with HTTP/2.

HTTP/3 is unstable yet, so I'm fine with no docs for them right now. But it'd be better if the sentence is something like TBD or nothing.

Reply:
I just remember the reason. I don't think the settings work now.

Reply:
I slightly changed the sentence. Feel free to change it if it's still not enough.